### ISSUE TITLE
env_broad_scale terms for East River watershed (study id gold:Gs0135149) aren't rooted in ENVO biome

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#611]
These came up in a query written by @turbomam. We need to do the following:
- [x] decide on a env_broad_scale term for these samples
- [ ] If different than GOLD let them know so they can update the values in GOLD

related to previous discussions from Oct 2021 
https://github.com/microbiomedata/nmdc-server/issues/537#issuecomment-941632979

**Comment by aclum** [comment]
Per Chris, Mark will add 'meadow biome' to ENVO to match the parallel 'meadow ecosystem' and do a release. Once this is done Alicia will communicate with GOLD.

**Comment by ssarrafan** [comment]
@aclum @turbomam is this required for GSP?

**Comment by turbomam** [comment]
I'll let @aclum answer the timing/priority

I'm am working on this now

**Comment by turbomam** [comment]
```sparql
PREFIX gold: <https://bioregistry.io/gold:>
PREFIX MIXS: <https://w3id.org/mixs/>
PREFIX nmdc: <https://w3id.org/nmdc/>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
select 
distinct  ?ebs_term ?ebs_label ?els_term ?els_label ?em_term ?em_label 
where {
    ?bs a nmdc:Biosample ;
        <http://purl.org/dc/terms/isPartOf> gold:Gs0135149 ;
        MIXS:0000012 ?ebs ;
        MIXS:0000013 ?els ;
        MIXS:0000014 ?em .
    ?ebs nmdc:term ?ebs_term .
    ?ebs_term rdfs:label ?ebs_label .
    ?els nmdc:term ?els_term .
    ?els_term rdfs:label ?els_label .
    ?em nmdc:term ?em_term .
    ?em_term rdfs:label ?em_label .
}
```

ebs_term | ebs_label | els_term | els_label | em_term | em_label
-- | -- | -- | -- | -- | --
ENVO:00000108 | meadow ecosystem | ENVO:00000292 | watershed | ENVO:00005802 | bulk soil

**Comment by turbomam** [comment]
```sparql
PREFIX gold: <https://bioregistry.io/gold:>
PREFIX dcterms: <http://purl.org/dc/terms/>
select * where {
    gold:Gs0135149 dcterms:description ?d .
}
```

> This research project aimed to understand how snow accumulation and snowmelt influences the mobilization of nitrogen through the soil microbiome in a mountainous catchment at the East River Watershed in Colorado. This project sought to identify bacteria, archaea, and fungi that were associated with the microbial biomass bloom that occurs during winter and the biomass crash following snowmelt. This project also sought to understand whether the traits that govern microbial community assembly during and after snowmelt were phylogenetically conserved. Samples were collected during winter, the snowmelt period, and after snowmelt in spring, from an area that transitioned from an upland hillslope to a riparian floodplain. This project is part of the Watershed Function Science Focus Area.

**Comment by turbomam** [comment]
LLM prompt:

> my colleagues want to describe the samples included in a scientific study. We use terms form the MIxS standard and the environment ontology, EnvO.
> 
> They want to say that the env_broad_scale (MIXS:0000012) value is 'meadow ecosystem' (ENVO:00000108), the env_local_scale (MIXS:0000013) value is 'watershed' (ENVO:00000292) and the env_medium (MIXS:0000014) value is 'bulk soil' (ENVO:00005802)
> 
> Here are the definitions of the MIxS terms:
> 
> env_broad_scale, "broad-scale environmental context": Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO s biome class: http://purl.obolibrary.org/obo/ENVO\_00000428. 
> 
> env_local_scale, "local environmental context": Report the entity or entities which are in the sample or specimen s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field.
> 
> env_medium, "environmental medium": Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483).  Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top)
> 
> EnvO documentation about how to use those MIxS fields: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
> 
> I object to that because "meadow ecosystem" isn't a subclass of biome as recommended above.
> 
> The study also provides this additional description: "This research project aimed to understand how snow accumulation and snowmelt influences the mobilization of nitrogen through the soil microbiome in a mountainous catchment at the East River Watershed in Colorado. This project sought to identify bacteria, archaea, and fungi that were associated with the microbial biomass bloom that occurs during winter and the biomass crash following snowmelt. This project also sought to understand whether the traits that govern microbial community assembly during and after snowmelt were phylogenetically conserved. Samples were collected during winter, the snowmelt period, and after snowmelt in spring, from an area that transitioned from an upland hillslope to a riparian floodplain. This project is part of the Watershed Function Science Focus Area."
> 
> please recommend EnvO class ids that we can use in those three MIxS fields. It would be good if we could retain the meadow and watershed knowledge in one of those fields, but I want to me more rigorous about the env_broad_scale as a a biome.

**Comment by turbomam** [comment]
LLM response:

> Thank you for providing the detailed context and the MIxS term definitions. Based on the information you've shared, here are my recommendations for the EnvO class IDs that align with the MIxS fields and the study description:
> 
> env_broad_scale (MIXS:0000012):
> 'montane biome' (ENVO:01000243) - This term is a subclass of 'biome' and captures the broad-scale environmental context of the study, which is a mountainous catchment.
> env_local_scale (MIXS:0000013):
> 'watershed' (ENVO:00000292) - As mentioned in the study description, the research focuses on the East River Watershed in Colorado. This term fits the definition of env_local_scale as it provides a more specific local environmental context.
> 'meadow' (ENVO:01000320) - To retain the meadow knowledge, you can include this term as it represents a smaller spatial grain than the montane biome and is relevant to the study area.
> 'riparian zone' (ENVO:00000384) - The study mentions that samples were collected from an area transitioning from an upland hillslope to a riparian floodplain. Including this term provides additional local environmental context.
> env_medium (MIXS:0000014):
> 'bulk soil' (ENVO:00005802) - This term is a subclass of 'environmental material' and directly relates to the study's focus on the soil microbiome and nitrogen mobilization through soil.
> 'snowpack' (ENVO:01001434) - Since the study investigates the influence of snow accumulation and snowmelt on the soil microbiome, including this term provides relevant information about the environmental material surrounding the samples.
> In summary, I suggest using the following EnvO class IDs for the MIxS fields:
> 
> env_broad_scale: 'montane biome' (ENVO:01000243)
> env_local_scale: 'watershed' (ENVO:00000292), 'meadow' (ENVO:01000320), 'riparian zone' (ENVO:00000384)
> env_medium: 'bulk soil' (ENVO:00005802), 'snowpack' (ENVO:01001434)
> This combination of terms captures the broad-scale biome context, retains the meadow and watershed knowledge, and provides additional relevant information about the local environment and the medium surrounding the samples, all while adhering to the MIxS field definitions and the EnvO ontology.
>

**Comment by turbomam** [comment]
TL;DR: is 'montane biome' (ENVO:01000243) an acceptable existing EnvO term for the broad-scale environmental context

Note that Claude is recommending that we use multiple local environmental context values. @cmungall also suggested that recently. I feel like MIxS provided mixed messages about which of the environmental context fields are multivalued. At this time, we have all of them set to single-valued in teh nmdc-schema

**Comment by turbomam** [comment]
Do we want to resume the bulk MIxS/EnvO to GOLD mapping effort that Bill started about 1.5 years ago?

**Comment by aclum** [comment]
@turbomam  you can get the info from the GOLD UI
from [this page](https://gold.jgi.doe.gov/biosamples?Study.GOLD+Study+ID=Gs0135149&Study.Study+Name=&Study.Add+Date=) you'd click on 'Select Columns for Table', expand the Environmental Information section and select 'Broad-scale Environmental Context', 'Local Environmental Context','Environmental Medium'. You can view this w/o being logged in but to download it you need to be logged in.

it would be valuable to resume GOLD ecosystem to ENVO mapping b/c this is a big barrier for sample ingest b/c NMDC requires it. We have some studies of interest where we could pilot this.

Based on the elevation listed in GOLD, 2780, compared to the surrounding areas as viewed on a topological map view this doesn't seem like a mountain area. I have strong preference to use what GOLD has, 'grassland biome [ENVO:01000177]', so we can close this issue out.

**Comment by ssarrafan** [comment]
Based on @aclum last comment assuming this is active.  I'll move to next sprint @turbomam.

**Comment by turbomam** [comment]
I think somebody just has to make a changesheet for the `env_broad_scale` value for all of the `Biosample`s that are associated with this `Study`. I agree with @aclum's decision about using 'grassland biome [ENVO:01000177]'

**Comment by aclum** [comment]
Since you and Chris are the ones spearheading this effort the request is that you issue the changesheet.

**Comment by turbomam** [comment]
Should I do this now or after GSP/ECR?

**Comment by ssarrafan** [comment]
> Should I do this now or after GSP/ECR?

It sounds like you should be able to do it now. Right? @aclum

**Comment by aclum** [comment]
It should be fine to do this now

**Comment by ssarrafan** [comment]
@turbomam is discussing with Chris this afternoon. Said to move to next sprint.  @aclum FYI

**Comment by ssarrafan** [comment]
@turbomam is almost done. Needs @aclum to sign off to complete. Moving to next sprint.

**Comment by turbomam** [comment]
I ran a changesheet last night. It looked like it was accepted. I'll check now.

**Comment by ssarrafan** [comment]
Moving to new sprint due to re-IDing timeline

**Comment by aclum** [comment]
@turbomam this project was re-id'd so you should be able to update the changesheet and submit now.

**Comment by ssarrafan** [comment]
@turbomam @aclum any update on this?  Done today?  Next sprint?  Backlog?

**Comment by ssarrafan** [comment]
Removing from sprint, no response on status, no updates for two weeks

**Comment by aclum** [comment]
changesheets has been applied to dev and prod. No need to update GOLD since we are using the same terms.
